### PR TITLE
Fix: Remove all Remaining Bugs in the Project

### PR DESCRIPTION
**Pull request by mhenriette**
## Describe your changes
removed repetitions and spaces that were not needed in the project sections
## Issue ticket number and link
- ID: 025
- Link:[Notion](https://www.notion.so/apeunit/bugs-in-all-sections-50769cd39b764aeab5a5ee3e08601fc3)

## Tasks completed
- [x] fixed the links in footer so that they are opened in new tab 
- [x] added alt name in hero section images 
- [x]  make Unit A bold
- [x] remove unnecessary spaces in sections
## Tasks not completed
none

### CODE REVIEW

File: components/ui/Title.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import {H2} from '../ui/Headings'
 const Title = ({content, styles}) => {
     return (
-        <div className={`bg-center text-white  bg-hero-pattern bg-no-repeat max-w-max ${styles}`}>
+        <div className={`bg-center text-white bg-hero-pattern bg-no-repeat max-w-max ${styles}`}>
             <H2 text={content} styles='pb-11.75 '></H2>
```
Comment (alain-kubwayo) [inline components/ui/Title.jsx:5]: The space in the class names passed into `styles` prop should be removed.

File: components/cards/WhatWeDoCard.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { H4 } from '../ui/Headings';
 
 const WhatWeDo = ({index, title, description, image}) => {
     return (
-        <div className='flex flex-col md:flex md:flex-row md:justify-between  gap-8 md:grid-cols-2 py-8 md:py-14.25 md:mt-12.5'>
+        <div className='flex flex-col md:flex md:flex-row md:justify-between gap-8 md:grid-cols-2 py-8 md:py-14.25 md:mt-12.5'>
             <div className={`md:w-3/4 lg:w-153.5 xl:w-141.25 space-y-4 order-2 ${index % 2 !== 0 ? 'md:order-2 ' : 'md:order-1'}`}>
```
Comment (alain-kubwayo) [inline components/cards/WhatWeDoCard.jsx:7]: Unnecessary white space after `md:order-2` class should be removed.

File: components/cards/Work.jsx [hunk 3]
```diff
@@ -6,17 +6,17 @@ import { DefaultButton } from "../ui/Button";
 const Work = ({ text, title, imageSrc2 }) => {
   return (
     <div className="pt-6 pb-8 xl:pt-32.75 xl:pb-27.5 bg-primary-800 w-full xl:w-320 rounded-xl flex-col lg:flex lg:flex-row md:justify-between px-4 xl:px-0 xl:pl-12.75 ">
```
Comment (alain-kubwayo) [inline components/cards/Work.jsx:8]: consider removing white space after `xl:pl-12.75` class

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good job, I left some comments.

File: components/ui/Title.jsx (hunk 1 above)
Comment (mhenriette) [reply components/ui/Title.jsx]: thanks for pointing out, it's resolved!!

File: components/cards/WhatWeDoCard.jsx (hunk 2 above)
Comment (mhenriette) [reply components/cards/WhatWeDoCard.jsx]: thanks for pointing out, it's resolved!!

File: components/cards/Work.jsx (hunk 3 above)
Comment (mhenriette) [reply components/cards/Work.jsx]: thanks for pointing out, it's resolved!!

**Review by alain-kubwayo — APPROVED** [review]

**Review by Delyc — APPROVED** [review]
Looks good!

**Pull request merged**